src/GT.Core/Services/Impl/GTListingService.cs

```diff
@@ -191,17 +192,17 @@ public async Task<List<ListingOverviewDTO>> GetAsync(IListingFilterModel? filter
 					filter.Location == null
 					|| (e.Location != null
 						&& e.Location.Name != null
-						&& string.Equals(filter.Location, e.Location.Name)))
+						&& EF.Functions.Like(e.Location.Name, filter.Location)))
 
 				.Where(e =>
 					filter.SalaryMin == null
-					|| (e.SalaryMax != null
-						&& e.SalaryMax >= filter.SalaryMin))
+					|| (e.SalaryMin != null
+						&& e.SalaryMin >= filter.SalaryMin))
```

Comment:
Salary min should be
 `filter.SalaryMin <= e.SalaryMax`

Reply:
Shouldn't this part of the filter return all listing entities where the minimum expected salary is greater than the minimum salary given by the filter? i.e. submitting a minimum salary value of 20000 (filter.SalaryMin) should return all listings where the lowest possible salary value (entity.SalaryMin) is greater than or equal to 20000.

It's possible that I've misunderstood the purpose of this part of the filter, maybe it would be a good idea to have a short meeting about it after DSTUM on 06/04-22?

Reply:
See my comment on next item. Same applies here.

Reply:
Answered this discussion in the comments of the next item below.